marklogic-data-hub/src/main/resources/ml-modules/services/mlJobs.sjs

```diff
@@ -36,19 +36,22 @@ function get(context, params) {
   else if(fn.exists(status)) {
     resp = datahub.jobs.getJobDocs(status);
   }
-  else if (fn.exists(flowNames)) {
-    resp = datahub.jobs.getJobDocsForFlows(flowNames);
-  }
-  else if (fn.exists(flow)) {
-    resp = datahub.jobs.getJobDocsByFlow(flow);
-  }
   else if (fn.exists(latest)) {
-    flowNames = (fn.exists(flowNames)) ? datahub.hubUtils.normalizeToSequence(flowNames) : flowNames;
+    flowNames = (fn.exists(flowNames)) ? datahub.hubUtils.normalizeToSequence(flowNames) : datahub.hubUtils.normalizeToSequence([flow]);
```

Comment:
I'm thinking a test can be added easily here that would run against JobDocManager. On the other hand, there are currently zero tests that hit JobDocManager directly. 

And - I just searched for uses of "latest", and I found none. JobDocManager never passes this as a parameter. So this looks like dead code?? I'm confused.

Reply:
There are a few latest uses in the JobDocManager, like com/marklogic/hub/job/JobDocManager.java:106 `params.put("latest", "true");`. I'll add a test for it.

Reply:
Are we looking at different things? I'm looking at https://github.com/marklogic/marklogic-data-hub/blob/develop/marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobDocManager.java on the develop branch, and there's no mention of "latest" there.

Reply:
Ah - I see it now, separate branch - https://github.com/marklogic/marklogic-data-hub/blob/feature/DHFPROD-3922/marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobDocManager.java